src/unity/toolkits/drawing_recognition/data_preparation.cpp

```diff
@@ -0,0 +1,382 @@
+/* Copyright © 2019 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+ */
+#include <time.h>
+#include <limits.h>
+#include <math.h>
+#include <iomanip>
+#include <logger/assertions.hpp>
+#include <image/image_type.hpp>
+#ifdef __APPLE__
+#include <CoreGraphics/CoreGraphics.h>
+#endif // __APPLE__
+#include "data_preparation.hpp"
+#include "random/random.hpp"
+#include "util/sys_util.hpp"
```

Comment:
It would be nice to clean up the list of includes. For example, it can be helpful to group these into system headers versus Turi headers. It seems weird to use angle brackets for some Turi headers and quotes for others, especially for those that don't live in the same directory as this file